test/wTabs.test.js

```diff
@@ -29,7 +29,7 @@ describe('tabs', () => {
       setup: () => ({ model })
     }
     const wrapper = mount(TabFixture)
-    assert.equal(wrapper.text(), 'Foo Bar OMG')
+    assert.equal(wrapper.text(), 'Foo BarOMG')
```

Comment:
This change is a result of a fix of `text()` method trimming text in elements. https://github.com/vuejs/test-utils/releases/tag/v2.4.2